.env.example

```diff
@@ -17,6 +17,7 @@
 ## FRONT-END URLS
 #ACTIVATION_URL=https://your_frontend_activation_url/{{token}}
 #FORGOT_PASSWORD_URL=https://your_frontend_password_reset_url/{{token}}
+#RETREAT_INVITATION_URL=https://example.com/retreat_invitation/{{token}}
```

Comment:
Please update the `CHANGELOG` so we don't forgot to update the ENV variable of the production system